src/Generator/Template/PropertyTemplate.php

```diff
@@ -11,13 +12,15 @@ class PropertyTemplate implements TemplateInterface
 {
     use VisibilityTrait;
 
-    const TEMPLATE = '
+    private const TEMPLATE = '
         %s %s $%s;
     ';
 
     /**
-     * @param \ReflectionProperty $property
+     * @param \Reflector|\ReflectionProperty $property
```

Comment:
Same as above.

Reply:
Because we call a specific method of the concrete class implementation.